inst

```diff
@@ -338,4 +341,38 @@ nextcloud_curl() {
     echo "$result"
 }
 
+detect_collabora () {
+    # When Collabora (CODE) or Collabora Online is already installed on the UCS system when Nextcloud is going to be installed,
+    # the richdocuments app is enabled and the WOPI URL is configured to the local system, if not already configured.
+    # If the app is not present, the richdocuments app is disabled
+    FQDN="$(ucr get hostname).$(ucr get domainname)"
+    echo "Check for richdocuments app"
+    if [ "$(ucr get appcenter/apps/collabora/status)" = "installed" ] || [ "$(ucr get appcenter/apps/collabora-online/status)" = "installed" ] ; then
+        univention-app shell nextcloud sudo -u www-data /var/www/html/occ app:enable richdocuments
+        occCode=$?
+        if [ "$(univention-app shell nextcloud sudo -u www-data /var/www/html/occ config:app:get richdocuments wopi_url)" = "" ] && [ ${occCode} -eq 0 ] ; then
+            univention-app shell nextcloud sudo -u www-data /var/www/html/occ config:app:set richdocuments wopi_url --value="https://$FQDN/"
+        fi
+    else
+        univention-app shell nextcloud sudo -u www-data /var/www/html/occ app:disable richdocuments
```

Comment:
I would not disable it to avoid false positives. There is a chance that an external office server is in use.

Reply:
The use case here is that Nextcloud is in the process of being installed. If an external office server should be used, it has to be configured manually directly in Nextcloud.

If a document server is already present on the same system, this configuration is done automatically.

If no document server is used locally, it is deactivated. If not, the app is there and it offers the user to open office files. This won't work without a document server and a "Not found" (app uninstalled) or "Service unavailable" (app stopped) error is shown to the user in Nextcloud.

I believe it is better to let the user manually reactivate the office app, if the document server is remote, rather than confronting the user with an ugly error message.

Reply:
Unless something has changed, it is possible that an update routine is being run (we check it an set a flag with `IS_UPDATE`). In that case, the admin should not need to re-enable it. And otherwise it is not enabled either.

Reply:
I fear I don't understand your point enough to add appropriate code.

Reply:
Let me rephrase:

1. The default state is that the app is disabled
2. This routine will be entered on Nextcloud app update, too, not only installation
3. → disabling is not necessary, but could disable an intentionally enabled app → bad UX
=> let's just drop this line of code

Reply:
All right. I just made a new commit and dropped the disable lines.